setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
```

Comment:
What is this setting up?

Reply:
Talked offline to explain the need for setup.py